app.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const app = express();
+const {getAllTopics} = require('./controllers/topics_controller');
+const {
+    handleCustomErrors,
+    handlePsqlErrors,
+    handleServerErrors,
+  } = require('./errors/index.js');
+
+app.get('/api/topics', getAllTopics)
+
+app.all('/*', (request, response) => {
+    response.status(404).send({msg: "404 Not Found"});
+});
+
+app.use(handleCustomErrors);
+app.use(handlePsqlErrors);
```

Comment:
Great thinking ahead of error handling. We don't need to handle the customErrors or psql errors at this point tho. Can always bring them back later when we need them (in those related tickets)